ci-operator/config/openshift/cluster-version-operator/openshift-cluster-version-operator-main.yaml

```diff
@@ -133,6 +133,28 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: openshift-e2e-azure
+- always_run: false
+  as: e2e-agnostic-operator-devpreview
+  optional: true
+  steps:
+    cluster_profile: azure4
+    test:
+    - as: e2e-test
+      cli: latest
+      commands: |
+        oc patch FeatureGate cluster --type='json' -p '[{"op": "replace", "path": "/spec/featureSet", "value":"DevPreviewNoUpgrade"}]'
+        oc wait --for create ns/openshift-update-status-controller --timeout=120s
+        oc -n openshift-update-status-controller wait --for create deployment/update-status-controller --timeout=30s
+        oc create cm -n openshift-update-status-controller status-api-cm-prototype
+        oc -n openshift-update-status-controller wait --for create cm/status-api-cm-prototype --timeout=10s
+        sleep 120
+        oc get cm -n openshift-update-status-controller status-api-cm-prototype -o yaml > ${ARTIFACT_DIR}/status-api-cm-prototype.yaml
```

Comment:
Actually, can we have the actual code steps in a script somewhere in CVO and just call that script from here?

Reply:
https://github.com/openshift/cluster-version-operator/pull/1142